examples/addition_rnn.py

```diff
@@ -193,11 +193,10 @@ class colors:
         q = ctable.decode(rowx[0])
         correct = ctable.decode(rowy[0])
         guess = ctable.decode(preds[0], calc_argmax=False)
-        print('Q', q[::-1] if INVERT else q)
-        print('T', correct)
+        print('Q', q[::-1] if INVERT else q, end=" ")
```

Comment:
Use `'` as string delimiter for consistency

Reply:
Done.